airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/__init__.py

```diff
@@ -52,32 +59,40 @@
 from metadata_service.constants import METADATA_FILE_NAME, METADATA_FOLDER
 
 ASSETS = [
```

Comment:
I don't really get why we need to explicitely declare the list of assets to make available in the orchestrator. Can't Dagster auto discover all our assets?

Reply:
Im sure this is possible. Will take the time to do this in a separate PR as this ones getting big.